tools/doxygen/Makefile

```diff
@@ -18,7 +18,7 @@ ifeq ($(HOST_OS),Windows)
   endif
 endif
 
-export docdirs = $(sort $(foreach dir,$(basedirs),${shell find $(contiking)/${dir} -type d}))
+export docdirs = $(sort $(foreach dir,$(basedirs),${shell find $(contiking)/${dir} -maxdepth 7 -type d}))
```

Comment:
@joakimeriksson Instead of using -print0. I set a limit to maxdepth of 7.
This should be more than enough, at this depth it is only submodules. At least, from what I could find in my investigations.

Reply:
Is the option suggested by @pjonsson below a possible solution - e.g. do a recursive config on doxygen?